app/move/steps/create.js

```diff
@@ -122,7 +123,7 @@ module.exports = {
   '/save': {
     skip: true,
     controller: Save,
-    next: 'document',
+    next: FEATURE_FLAGS.DOCUMENTS ? 'document' : 'final',
```

Comment:
Should we have a test around the feature flag on/off behaviour?

Reply:
I was even wondering if this was the best place for it. Because I didn't then feel comfortable testing a config file.

I might move this to the controller actually, that way it can be tested more neatly.